src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -259,16 +259,21 @@ def suppress_one_group_saturated_ramps(ramp_data):
     dq = ramp_data.groupdq
     nints, ngroups, nrows, ncols = dq.shape
     sat_flag = ramp_data.flags_saturated
+    jump_flag = ramp_data.flags_jump_det
```

Comment:
May want to change the name (and perhaps the description) of this function since this is not strictly for saturation.

Reply:
I changed the function name.